templates/focus-ios.xliff

```diff
@@ -381,6 +381,42 @@
         <source>Close</source>
         <note>Button label used to close a menu that displays as a popup.</note>
       </trans-unit>
+      <trans-unit id="Onboarding.Button.Title">
+        <source>Start browsing</source>
+        <note>Text for a label that indicates the title of button from onboarding screen</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.History.Description">
+        <source>Erase your browsing history, passwords, cookies, and prevent unwanted ads from following you in a simple click!</source>
+        <note>Text for a label that indicates the description of history section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.History.Title">
+        <source>Your history doesn’t follow you</source>
+        <note>Text for a label that indicates the title of history section from onboarding screen.</note>
+      </trans-unit>
+      <trans-unit id="Onboarding.Incognito.Description">
+        <source>Focus is a dedicated privacy browser with tracking protection and content blocking.</source>
```

Comment:
"Focus" should be a placeholder since it also can be "Klar"

Reply:
corrected here: https://github.com/mozilla-l10n/focusios-l10n/pull/80/files#diff-5aaa6e083e6c89a11d87391ecf7ab52b669793c5ffc9c713b2471b628fb04959R462-R463